hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsDatanodeService.java

```diff
@@ -210,6 +212,18 @@ public void start(OzoneConfiguration configuration) {
   }
 
   public void start() {
+    serviceRuntimeInfo = new DNMXBeanImpl(HddsVersionInfo.HDDS_VERSION_INFO) {
+      @Override
+      public String getNamespace() {
+        String localScmServiceId = conf.getTrimmed(
+            ScmConfigKeys.OZONE_SCM_DEFAULT_SERVICE_ID);
+        if (localScmServiceId == null) {
+          Collection<String> scmServiceIds = conf.getTrimmedStringCollection(OZONE_SCM_SERVICE_IDS_KEY);
+          return String.join(",", scmServiceIds);
+        }
+        return localScmServiceId;
```

Comment:
Call existing method to determine the single service that the datanode belongs to:

https://github.com/apache/ozone/blob/4670a5ef8c98ae236d1acbaf93e9609309f88094/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAUtils.java#L172-L192

to avoid duplication and mismatch (in case of change).